src/components/toast/toast.ts

```diff
@@ -36,6 +36,13 @@ export default class IgcToastComponent extends LitElement {
   @property({ type: Boolean, reflect: true, attribute: 'keep-open' })
   public keepOpen = false;
 
+  constructor() {
+    super();
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'alert');
+    }
+  }
```

Comment:
You should move this to the connectedCallback hook.
As per the [specification](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-element-conformance):

> The element must not gain any attributes or children, as this violates the expectations of consumers who use the [createElement](https://dom.spec.whatwg.org/#dom-document-createelement) or [createElementNS](https://dom.spec.whatwg.org/#dom-document-createelementns) methods.